ArchiSteamFarm/Commands.cs

```diff
@@ -1813,6 +1825,160 @@ public sealed class Commands {
 			return responses.Count > 0 ? string.Join(Environment.NewLine, responses) : null;
 		}
 
+		private string? ResponseMatchActivelyBlacklist(ulong steamID) {
+			if ((steamID == 0) || !new SteamID(steamID).IsIndividualAccount) {
+				throw new ArgumentOutOfRangeException(nameof(steamID));
+			}
+
+			return !Bot.HasAccess(steamID, BotConfig.EAccess.Master) ? null : FormatBotResponse(Bot.BotDatabase.MatchActivelyBlacklistedAppIDs.Count == 0 ? string.Format(CultureInfo.CurrentCulture, Strings.ErrorIsEmpty, nameof(Bot.BotDatabase.MatchActivelyBlacklistedAppIDs)) : string.Join(", ", Bot.BotDatabase.MatchActivelyBlacklistedAppIDs));
```

Comment:
Can we make this more readable and not nest the ternary conditional operator? This would be the first instance in the repository. We also have the `if (!Bot.HasAccess(...)) { return null; }` explicitly everywhere else.

proof for it being the first instance 😉
![image](https://user-images.githubusercontent.com/6608231/107965768-2ffafa00-6fab-11eb-8269-791c1a88bcfd.png)

Reply:
> This would be the first instance in the repository

It's basically a copypaste from `ResponseIdleBlacklist` method: https://github.com/JustArchiNET/ArchiSteamFarm/blob/main/ArchiSteamFarm/Commands.cs#L1261

> proof for it being the first instance

Then make sure your proofs are really proofs 😉